.ci/no-exception-test.sh

```diff
@@ -53,6 +53,35 @@ guava-with-sun-checks)
   rm sun_checks.*
   ;;
 
+openjdk14-with-checks-nonjavadoc-error)
+  CS_POM_VERSION=$(mvn -e -q -Dexec.executable='echo' -Dexec.args='${project.version}' \
+                     --non-recursive org.codehaus.mojo:exec-maven-plugin:1.3.1:exec)
+  echo CS_version: $CS_POM_VERSION
+  mkdir -p .ci-temp/
+  cd .ci-temp/
+  # activate below line after https://github.com/checkstyle/contribution/pull/516
+  # git clone https://github.com/checkstyle/contribution
+  # need to remove below line after https://github.com/checkstyle/contribution/pull/516
+  git clone https://github.com/nmancus1/contribution.git
```

Comment:
Remaining travis failure is due to spellcheck failing on "nmancus", which will be removed after https://github.com/checkstyle/contribution/pull/516